### DISCUSSION TITLE
Data automatically deleted from codespace - "polling-app"

### DISCUSSION THREAD

**Discussion by codeNitesh** [community/community#40811] [Codespaces]
### Select Topic Area

Bug

### Body

Hi team,
@github-staff 

I was working with codespaces and was really happy with the feature. But I don't know why, my data from the app - "polling-system" automatically deleted after my push to the github repo. The data inside folder "FRONTEND" got deleted.
Please restore it quickly, I was working on it so hard since last couple of days. It is very important college work, very important for me.

Kindly restore it... It is my request.

**Comment by eljog** [comment]
Hi @codeNitesh 
From what I understand the data was deleted from the repo after git push. Was the data available on your branch before the last push? In that case you will be able to see the commits that deleted the data in repo history. 

Please let me know if that helps.
